### PR TITLE
Team update

### PR DESCRIPTION
**Pull request by jburel**
Move roger to former member
cc @sbesson @joshmoore

### CODE REVIEW

File: teams/index.html [hunk 1]
```diff
@@ -115,8 +115,7 @@ <h4 class="text-center">Jean-Marie Burel</h4>
                       <p>
                           Jean-Marie Burel joined the staff of the Swedlow lab
                           in 2003. Since then, he's been contributing to the
-                          development of OME, and spends most of his time
-                          working on the OMERO.insight project. He received is
+                          development of OME. He received is
```
Comment (joshmoore) [inline teams/index.html]: received _his_ ?

Comment (jburel) [reply teams/index.html]: typo has been there a many years
Thanks for spotting it

File: teams/index.html [hunk 2]
```diff
@@ -504,32 +504,32 @@ <h4 class="text-center">Petr Walczysko</h4>
...
                         <span class="bio-link tiny button hollow">Read Bio</span>
                     </a>
-                    <div class="reveal" id="modal20" data-reveal>
+                    <div class="reveal" id="modal18" data-reveal>
                       <h4 class="text-center">Frances Wong</h4>
                       <p class="job-title">Curator</p>
                       <hr>
                       <p>
-                          
+                          Frances Wong joined the OME team in September 2018 as an OME curator. She received her PhD in Developmental Biology from the University of Edinburgh. She then carried out post-doctoral research in molecular genetics and biomedical imaging. She also worked on atlas-based gene expression databases and online scientific resources.
```
Comment (mtbc) [inline teams/index.html]: maybe second "OME" is superfluous

**Comment by jburel** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/teams/

**Comment by joshmoore** [comment]
Barring the very long lines (which can be unified later) all looks good to me. An idea for the former member section would be to have the years they were with us. (I could also see linking to their github as a way of uniquely identifying the person)

:+1:

**Comment by mtbc** [comment]
That's an idea: yes, for alumni years would be a good addition. (Outwith this PR.)

**Comment by jburel** [comment]
I will look into this idea of GH account after
Merging so it can upgrade as part of the 5.4.8 release

**Pull request merged**